4 - AnimalShelter/Code/AnimalType.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace AnimalShelterCode
+{
+    public enum AnimalType
+    {
+        Cat,
+        Dog,
+        Deer,
+        Bear,
+        Bird,
+        Snake
+    }
```

Comment:
Having an enum for the "type" of animal does work but note this is considered a "rigid" design.